lib/src/widgets/country_picker/country_selector.dart

```diff
@@ -1,22 +1,60 @@
 import 'package:flutter/material.dart';
-import 'package:phone_form_field/src/helpers/country_finder.dart';
-import 'package:phone_form_field/src/models/all_countries.dart';
-import 'package:phone_form_field/src/widgets/country_picker/search_box.dart';
 
+import '../../helpers/country_finder.dart';
+import '../../localization/phone_field_localization.dart';
+import '../../models/all_countries.dart';
 import '../../models/country.dart';
 import 'country_list.dart';
+import 'search_box.dart';
+
+const _emptyFavCountriesArray = <String>[];
 
 class CountrySelector extends StatefulWidget {
+  /// List of countries to display in the selector
+  /// Value optional in constructor.
+  /// when omitted, the full country list is displayed
   final List<Country> countries;
+
+  /// Callback triggered when user select a country
   final Function(Country) onCountrySelected;
+
+  /// ListView.builder scroll controller (ie: [ScrollView.controller])
   final ScrollController? scrollController;
 
+  /// Sort the countries automatically by localized name.
+  /// Note that the favorites countries are not sorted but
+  /// displayed in defined order.
+  final bool sortCountries;
+
+  /// Determine the countries to be displayed on top of the list
+  /// Check [addSeparator] property to enable/disable adding a
+  /// list divider between favorites and others defaults countries
+  final List<String> favoritesCountries;
+
+  /// Whether to add a list divider between favorites & defaults
+  /// countries.
+  final bool addSeparator;
```

Comment:
This is ambiguous. I'd rename to something that clearly states that the divider is to be placed between favorites and defaults.

Reply:
I don't find an appropriate property. My current idea is quite a long term  `addSeparatorAfterFavorites`... 

Any better idea would be greatly appreciated ;)

Reply:
addFavoritesSeparator should do it

Reply:
updated ;)